README.md

```diff
@@ -97,17 +97,22 @@ broken RPM build environment. You should add this to your ~/.rpmmacros:
 
 ## Creating WINDOWS INSTALLER
 
-Install Cygwin and include at least the ghostscript, zip, dos2unix and libgcc
+Install Cygwin and include at least the gcc, make, cygrunsrv, ghostscript, zip, dos2unix and libgcc
 packages.
 
 In case you are using a 64-bit version of Cygwin: rename cyggcc_s-1.dll to
 cyggcc_s-seh-1.dll in Makefile and win/setup.iss.
 
+Install Cygwin using
+
+    setup-x86_64.exe -qgnNdO -l C:\cygwin64\var\cache\setup -R C:\cygwin64 -s http://cygwin.mirror.constant.com -P gcc-core -P make -P ghostscript -P dos2unix -P zip -P cygrunsrv
+
```

Comment:
This is a bit opinionated. Options "n" and "N" disable creation of shortcuts, which contradict the instruction below. Probably the only relevant options are "-P" for installing packages.